src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -701,6 +701,33 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 				 }
 				}
 			}
+            if let exposureWindowsUntilTestResult = metadata?.exposureWindowsUntilTestResult {
```

Comment:
Indentation seems to be broken....perhaps only here in Github?

Reply:
Yes, GitHub issue